server/internal/core/domain/market_hour.go

```diff
@@ -0,0 +1,9 @@
+package domain
+
+type MarketHour struct {
+	ID              int64
+	FirstMarketHour int64
+	Period          int64
+	RoundLifetime   int64
+	CreatedAt       int64
```

Comment:
not needed

Reply:
I think it's valuable for clients (maybe another domain) to know which was the `upatedAt` moment, to safely know when to update his own cached settings.

Reply:
Done